lib/fluent/log.rb

```diff
@@ -350,8 +411,13 @@ def event(level, args)
       return time, message
     end
 
-    def caller_line(time, depth, level)
-      log_msg = "#{time.strftime(@time_format)}[#{LEVEL_TEXT[level]}]: "
+    def caller_line(type, time, depth, level)
+      worker_id_part = if type == :default && (@process_type == :worker0 || @process_type == :workers)
+                         @worker_id_part
+                       else
+                         ""
+                       end
+      log_msg = "#{time.strftime(@time_format)}[#{LEVEL_TEXT[level]}]: #{worker_id_part}"
```

Comment:
How about inserting `worker_id_part` before `:`?
It seems clear that this is system header, not message body.